src/bin/cargo/commands/run.rs

```diff
@@ -101,3 +105,16 @@ pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
         }
     })
 }
+
+pub fn is_manifest_command(arg: &str) -> bool {
```

Comment:
“Manifest commands” is already a term, though its definition is not pretty clear to me.

https://doc.rust-lang.org/nightly/cargo/commands/manifest-commands.html